db/migrate/20210202000626_add_tmdb_id_to_movies.rb

```diff
@@ -0,0 +1,6 @@
+class AddTmdbIdToMovies < ActiveRecord::Migration[5.2]
+  def change
+    add_column :movies, :tmdb_id, :bigint
+    add_index :movies, :tmdb_id, unique: true
+  end
```

Comment:
Since we're changing the movies migration to add the year, why not add the `tmdb_id` and `imdb_id` there and avoid these 2 migrations?